### PR TITLE
Update docs for beta.1 release

### LINKED ISSUE #3741: Revert the changes to 'Start-PSBuild' made in #3740 and #3742 after the beta.1 release
PR #3740 and #3742 made a hack to the build script to unblock the release. It needs to be reverted after the release.

### PR DESCRIPTION
**Pull request by daxian-dbw**
Update the docs for `beta.1` release.
Revert the build workarounds to fix #3741

### CODE REVIEW

**Comment by daxian-dbw** [comment]
Win7 packages are not available in beta.1 release due to a downlevel API set issue (#3747), so README.md still points to the alpha.18 win7 packages.

File: README.md [hunk 1]
```diff
@@ -32,19 +32,19 @@ You can download and install a PowerShell package for any of the following platf
...
-| macOS 10.11                        | [.pkg][rl-macos]       | [Instructions][in-macos]      |
+| macOS 10.12                        | [.pkg][rl-macos]       | [Instructions][in-macos]      |
 | Docker                             |                        | [Instructions][in-docker]     |
 
-[rl-windows10]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-win10-win2016-x64.msi
-[rl-windows81]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-win81-win2012r2-x64.msi
+[rl-windows10]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/PowerShell-6.0.0-beta.1-win10-win2016-x64.msi
+[rl-windows81]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/PowerShell-6.0.0-beta.1-win81-win2012r2-x64.msi
 [rl-windows7-64]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-win7-win2008r2-x64.msi
 [rl-windows7-86]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-win7-x86.msi
```
Comment (iSazonov) [inline README.md]: What about Win7? Remove link or update?

File: README.md [hunk 2]
```diff
@@ -32,19 +32,19 @@ You can download and install a PowerShell package for any of the following platf
...
-[rl-macos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/powershell-6.0.0-alpha.18-osx.10.11-x64.pkg
-[rl-opensuse421]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/powershell-6.0.0_alpha.18-1.suse.42.1.x86_64.rpm
+[rl-ubuntu16]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/powershell_6.0.0-beta.1-1ubuntu1.16.04.1_amd64.deb
+[rl-ubuntu14]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/powershell_6.0.0-beta.1-1ubuntu1.14.04.1_amd64.deb
+[rl-centos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/powershell-6.0.0_beta.1-1.el7.centos.x86_64.rpm
+[rl-ai]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/PowerShell-6.0.0-beta.1-x86_64.AppImage
+[rl-macos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/powershell-6.0.0-beta.1-osx.10.12-x64.pkg
+[rl-opensuse421]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.1/powershell-6.0.0_beta.1-1.suse.42.1.x86_64.rpm
```
Comment (iSazonov) [inline README.md]: Why links for SuSe and CentOS still use "_" -   "6.0.0_beta.1" ?

File: README.md (hunk 1 above)
Comment (daxian-dbw) [inline README.md]: Win7 is not supported in beta.1 release due to missing of api sets in .NET Core 2.0. We leave the pointers in README referring to alpha.18 win7 packages. Please see my comment above.

File: README.md (hunk 2 above)
Comment (daxian-dbw) [inline README.md]: Yeah, I guess they haven't been normalized in the package creation script yet.

File: README.md (hunk 1 above)
Comment (daxian-dbw) [inline README.md]: Note that the release log of beta.1 has a section about why win7 packages are not available.

Comment (iSazonov) [inline README.md]: I see that in release notes but not in the Read.me

Comment (daxian-dbw) [inline README.md]: @joeyaiello Maybe we should put the "win7 packages not available" in the [known issues](https://github.com/PowerShell/PowerShell/blob/master/docs/KNOWNISSUES.md).

**Review by TravisEz13 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (TravisEz13) [inline README.md]: @iSazonov  Can you file an issue?

Comment (iSazonov) [inline README.md]: Yes, tomorrow.

Comment (iSazonov) [inline README.md]: Ready #3760

**Pull request merged**